src/citrine/informatics/predictors/chemical_formula_featurizer.py

```diff
@@ -152,5 +153,23 @@ def __init__(self,
         self.excludes = excludes if excludes is not None else []
         self.powers = powers if powers is not None else [1]
 
+    @property
+    def powers(self) -> List[int]:
+        """The list of powers when computing generalized weighted means of element properties."""
+        warn("The type of 'powers' will change to a list of floats in v4.0.0. To retrieve them as "
+             "floats now, use 'powers_as_float'.")
+        return [int(p) for p in self._powers]
```

Comment:
```suggestion
        converted = [round(p) for p in self._powers]
        if converted == self._powers:
            warn("The type of 'powers' will change to a list of floats in v4.0.0. To retrieve them as "
                 "floats now, use 'powers_as_float'.", DeprecationWarning)
        else:
            diffs = [f"{x} => {y}" for x, y in zip(self._powers, converted) if x != y]]
            warn(f"The following powers were cast to ints: {'; '.join(diffs)}. To retrieve them as "
                 "floats, use 'powers_as_float'.")
        return converted
```

Reply:
I don't love adding an extra warning, but if it will move this PR forward, then I'm willing to do so.

Adding `round` does not preserve the old behavior, so it does not belong here.